Makefile

```diff
@@ -51,15 +50,14 @@ test: ## Run tests
 # build packages
 
 build: ## Build a release (sdist and wheel)
-	$(python) -m pip install --upgrade setuptools wheel
-	$(python) setup.py sdist bdist_wheel
+	$(python) -m pip install --upgrade build
+	$(python) -m build
 .PHONY: build
 
 test-release: build ## Test a built release
 	./test-release
 .PHONY: test-release
 
 release: ## Upload a built release
-	$(twine) upload dist/colorama-$(version)*{.whl,.tar.gz}
+	$(twine) upload dist/colorama-*{.whl,.tar.gz}
```

Comment:
Is it worth guarding against there being multiple matches for dist/*.whl, maybe by the 'build' step removing any old versions? This seems rampantly ad-hoc and racey and error-prone. Any other thoughts are welcome. (e.g. can we invoke anything here to get the specific version?)

Reply:
No need IMO because PyPI does not allow uploads that would overwrite existing artifacts